test/sms_nurse.test.js

```diff
@@ -28,6 +29,8 @@ describe("app", function() {
                     name: "sms_nurse",
                     testing_today: "2014-04-04 07:07:07",
                     testing_message_id: "0170b7bb-978e-4b8a-35d2-662af5b6daee",
+                    env: 'test',
+                    metric_store: 'test_metric_store',
```

Comment:
It seems like these lines have just been moved from lower down. Rather put them back where they were to minimise unnecessary changes that might have unexpected effects